tests/suit/framework/ova.go

```diff
@@ -0,0 +1,71 @@
+package framework
+
+import (
+	"context"
+	"os"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// LoadSourceDetails - Load Source VM details from ova
+func (r *OvaClient) LoadSourceDetails() (vm *OvaVM, err error) {
+
+	r.storageClass = DefaultStorageClass
+	if sc := os.Getenv("STORAGE_CLASS"); sc != "" {
+		r.storageClass = sc
+	}
+
+	r.vmData.testVMId = "b023ad5c55444c081c0837f5caa8dc3b2b9f"
+	r.vmData.testNetworkID = "ae1badc8c693926f492a01e2f357d6af321b"
+	r.vmData.testStorageName = "Dummy storage for source provider ova-provider"
+	return &r.vmData, nil
+}
+
+func (r *OvaClient) GetNfsServerForOva(k8sClient *kubernetes.Clientset) (string, error) {
+	storageClass, err := k8sClient.StorageV1().StorageClasses().Get(context.TODO(), r.storageClass, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	var nfsShare string
+	for parm, val := range storageClass.Parameters {
+		if parm == "server" {
+			nfsShare = nfsShare + val
+		}
+		if parm == "share" {
+			nfsShare = nfsShare + val
+		}
+	}
+	if nfsShare != "" {
```

Comment:
when would nfsShare be ""?

Reply:
If we ruining locally and the env variables wasn't set when running the CI setup, but we can drop it.

Reply:
yeah we need to return error if its empty, added in the code.